extensions/amp-story-panning-media/amp-story-panning-media.md

```diff
@@ -0,0 +1,159 @@
+    <amp-story-panning-media layout="fill" x="20%" y="30%" zoom="2">
+      <amp-img layout="fill" src="assets/world-map.jpg" width="4000" height="3059" ></amp-img>
+    </amp-story-panning-media>
+  </amp-story-grid-layer>
+</amp-story-page>
+```
+
+## Attributes
+
+### x (optional)
+
+Specifies the horizontal position in percentage.  
+Centered on 0 (default) with positive values moving the image to the right (50% centering left edge of image) and negative to the left (-50% centering right edge of image).
+
+### y (optional)
+
+Specifies the vertical position in percentage.
+Centered on 0 (default) with positive values moving the image down (50% centering top edge of image) and negative upward (-50% centering bottom edge of image).
+
+### zoom (optional)
+
+Specifies the level of zoom.
+The default is 1. This corresponds to the image fitting the viewport with [`layout=fill`](https://amp.dev/documentation/guides-and-tutorials/learn/amp-html-layout/?format=stories).
+A higher value scales the image up (zooms in). A lower value scales the image down (zooms out).
+
+### lock-bounds (optional)
+
+Prevents the image from panning beyond the viewport. When using `lock-bounds` `width` and `height` must be specified on the `amp-img` child.
+
+## Demos
+
+With these components we wanted to help users create more entertaining and immersive experiences, and to facilitate that, we have created demos with good use cases of the components. Feel free to ~~steal~~ implement any ideas from these demos into your own stories (or check the source code).
+
+<table>
+  <tr>
+   <td><strong>The Northern Sky Constellations:</strong> This story uses a star chart for the entire story by panning and zooming into different constellations. A parallax transition effect is also acheived by layering two png star images with transparency.
+<br><br>
+<a href="https://philipbell-panning-media.web.app/examples/amp-story/northern-sky-constellations.html">The Northern Sky Constellations</a>
+   </td>
+<td>
+<img src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/constellations-screenshot.jpg">
+   </td>
+  </tr>
+<tr>
+<td>
+<img src="https://raw.githubusercontent.com/processprocess/amphtml/panning-media-docs/extensions/amp-story-panning-media/img/controls.jpg">
+   </td>
+   <td>Story featuring sliders to modify the attributes of the panning-media component on the current page. This is an example of how a gui in a creation tool could configure the component.
+<br><br>
+<a href="https://philipbell-panning-media.web.app/examples/amp-story/controls.html">https://webstoriesinteractivity-beta.web.app/animals-polls.html</a>
+   </td>
+</tr>
+</table>
+
+You can check the roadmap in https://github.com/ampproject/amphtml/projects/121
+
+## FAQ
+
+**Can we have more than one component per page?**
+
+Yes, the components will automatically be associated with each other by sharing the same `src` value.
```

Comment:
This information can be moved under the example in configuration. I would rewrite it to say something like "You may include one or multiple amp-story-panning-media elements per amp-story-page..." then explain what is meant by "associated" (I'm guessing it means it pans to another area of the image?)